src/orquestra/sdk/_shared/schema/workflow_run.py

```diff
@@ -11,7 +11,8 @@
 import typing as t
 import warnings
 
-from .._dates import Instant
+from orquestra.sdk._shared.dates._dates import Instant
```

Comment:
**nitpick**: any particular reason to switch from a relative import to an absolute, and (overly) nested import?